app/Http/Controllers/Blogs.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class Blogs extends Controller
```

Comment:
It is good practice in Laravel to name controllers like `BlogController`. So, the singular model name and ending with ...Controller.